src/tools/genpolicy/src/yaml.rs

```diff
@@ -163,6 +164,14 @@ pub fn new_k8s_resource(
             debug!("{:#?}", &job);
             Ok((boxed::Box::new(job), header.kind))
         }
+        "CronJob" => {
+            let cronJob: cronjob::CronJob = serde_ignored::deserialize(d, |path| {
+                handle_unused_field(&path.to_string(), silent_unsupported_fields);
+            })
+            .unwrap();
+            debug!("{:#?}", &cronJob);
```

Comment:
Remove this?

Reply:
All other resource types in this block have a similar debug print. I'd say keep, may be useful for debugging (only prints if running with `RUST_LOG=debug`)